v2/call.js

```diff
@@ -298,6 +298,48 @@ function readUInt8String(frame, offset) {
     return fastBufferToString(frame.buffer, offset, end);
 }
 
+CallRequest.RW.lazy.readArg1Str = function readArg1Str(frame) {
+    if (frame.cache.arg1Str !== null) {
+        return frame.cache.arg1Str;
+    }
+
+    if (!frame.cache.csumStartOffset) {
+        var success = scanAndSkipHeaders(frame);
+        if (!success) {
+            return null;
+        }
+    }
+
+    var offset = frame.cache.csumStartOffset;
+
+    if (frame.size < offset + 1) {
+        return null;
+    }
+    var csumType = frame.buffer.readUInt8(offset, false);
+    offset += 1;
+
+    if (csumType !== 0x00) {
+        offset += 4;
```

Comment:
please use checksum utility function or simple lookup table by type for this, don't hardcode